lib/patches/bundler/bundler/fetcher/downloader.rb

```diff
@@ -1,21 +0,0 @@
-Truffle::Patching.require_original __FILE__
-
-if Truffle::Boot.patching_openssl_enabled?
-  # TruffleRuby: Use curl in bundler downloader for https requests
-
-  require "truffle/https_downloader"
```

Comment:
That file (`lib/patches/bundler/bundler/fetcher/downloader.rb`) can be removed too since those were the only 2 usages of it.